### ISSUE TITLE
Windows support?

### ISSUE THREAD

**Issue by apiszcz** [MobilityDB/PyMEOS#1]
Is the build expected to work with Windows?

![image](https://user-images.githubusercontent.com/1430861/189541274-af7bdc8d-4727-4dbc-bd64-7110c51751b3.png)

**Comment by Diviloper** [comment]
As of now, it is not.
Work to make it available for windows will start once MEOS is ready for Windows.
You can track the progress in the corresponding MEOS issue ([https://github.com/MobilityDB/MobilityDB/issues/176](https://github.com/MobilityDB/MobilityDB/issues/176)).

**Comment by anitagraser** [comment]
Still an issue in 2024

**Comment by Diviloper** [comment]
Indeed, there has not been much progress regarding this issue, since it's still blocked by MEOS not being available for Windows (other than through WSL).

I've created a new issue in the MobilityDB repo to track the evolution of Windows support for MEOS: https://github.com/MobilityDB/MobilityDB/issues/513

**Comment by anitagraser** [comment]
I see that https://github.com/MobilityDB/MobilityDB/issues/513 is closed now, does that mean we'll have pymeos on Win soon?
